### PR TITLE
Hotfix integration test

### PR DESCRIPTION
**Pull request by minsulee2**
My apology! This is the same issue as it's just fixed.
Ref: #1072

### CODE REVIEW

File: test/integration/he_sharding.test.js [hunk 1]
```diff
@@ -352,6 +352,7 @@ describe('HE Sharding', () => {
         server1_proc.kill();
         await waitForNewBlocks(server2, sharding.reporting_period * 3);
         console.log(`        --> Restarting server[0]...`);
+        ENV_VARIABLES[2].PEER_CANDIDATE_JSON_RPC_URL = "http://localhost:9092/json-rpc";
```
Comment (platfowner) [inline test/integration/he_sharding.test.js:355]: minor) what about deleting this env variable once the test is completed to minimize side effects (sharding.test.js case as well)?

Comment (minsulee2) [reply test/integration/he_sharding.test.js:355]: Done!

**Comment by minsulee2** [comment]
PTAL!

File: test/integration/he_sharding.test.js (hunk 1 above)
Comment (platfowner) [reply test/integration/he_sharding.test.js:355]: minor) oh, could this line be moved to 'before()' like deletion is done in 'after()'?

**Review by platfowner — APPROVED** [review]
LGTM with some suggestions.
Thanks for handling this issue!!

File: test/integration/he_sharding.test.js (hunk 1 above)
Comment (minsulee2) [reply test/integration/he_sharding.test.js]: Done!

**Comment by minsulee2** [comment]
Thanks for the fast review! Merging now!

**Pull request merged**